### DISCUSSION TITLE
Td-agent not able to install in customize path instead of default path

### DISCUSSION THREAD

**Discussion by Priyankanavalagund** [fluent/fluentd#4192] [Q&A]
I am trying to install td-agent in centos with below cmd
curl -L https://toolbelt.treasuredata.com/sh/install-redhat-td-agent4.sh | sh

But its installing default location /opt/td-agent/bin but i need to install customize path like this /AX/Release/ProjName/

while installing td-agent by default ruby also installed in default location. so ruby is pointing to default path. i need help how we can install td-agent in customize path.

Please help me as soon as possible. Thank you so much for your help

**Comment by kenhys** [comment]
Basically, customizing install path is not supported except for windows.

It's well known issue: https://github.com/fluent/fluent-package-builder/issues/488
